docs/install/macos.rst

```diff
@@ -14,7 +14,11 @@ Install
 #. Download the `.dmg installer <https://learningequality.org/download/>`_ for Kolibri **version 0.13**.
 #. Double-click the downloaded ``.dmg`` file.
 #. Drag the Kolibri logo inside the **Applications** folder, and wait for the installation to finish.
-#. Start the Kolibri app from the **Launchpad**.
+   
+   .. figure:: /img/copy-app.png
+     :alt: 
+
+4. Start the Kolibri app from the **Launchpad**.
```

Comment:
> Start the Kolibri app from the **Launchpad**, the **Applications** folder, or **Spotlight**

Reply:
Here, we should link to and summarize: https://support.apple.com/guide/mac-help/open-a-mac-app-from-an-unidentified-developer-mh40616/mac